src/image-copy/azext_imagecopy/cli_utils.py

```diff
@@ -86,3 +88,9 @@ def get_storage_account_id_from_blob_path(cmd, blob_path, resource_group, subscr
         namespace='Microsoft.Storage', type='storageAccounts', name=storage_account_name)
 
     return storage_account_id
+
+
+def format_cmd(cmd: List[str]):
+    if len(cmd) > 3:
+        return 'Running: az ' + ' '.join(cmd[3:])
+    return 'Running: ' + ' '.join(cmd)
```

Comment:
This may be confusing when `cmd`'s items contain spaces or quotes. Say `hello there` will become `az hello there`, instead of `az "hello there"`. `"hello there"` will become `az "hello there"`, instead of `az "\"hello there\""`.

Reply:
Yes. The returned string may not recover the intent of cmd variable.